Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -588,6 +588,48 @@ public DonchianChannel DCH(Symbol symbol, int period, Resolution? resolution = n
             return D(symbol, riskFreeRate, dividendYield, optionModel, ivModel, resolution);
         }
 
+        /// <summary>
+        /// Creates a new Delta indicator for the symbol The indicator will be automatically
+        /// updated on the symbol's subscription resolution
+        /// </summary>
+        /// <param name="symbol">The option symbol whose values we want as an indicator</param>
+        /// <param name="mirrorOption">The mirror option for parity calculation</param>
+        /// <param name="riskFreeRate">The risk free rate</param>
+        /// <param name="dividendYield">The dividend yield</param>
+        /// <param name="optionModel">The option pricing model used to estimate Delta</param>
+        /// <param name="ivModel">The option pricing model used to estimate IV</param>
+        /// <param name="resolution">The desired resolution of the data</param>
+        /// <returns>A new Delta indicator for the specified symbol</returns>
+        [DocumentationAttribute(Indicators)]
+        public Delta D(Symbol symbol, Symbol mirrorOption, decimal? riskFreeRate = null, decimal? dividendYield = null, OptionPricingModelType optionModel = OptionPricingModelType.BlackScholes, 
+            OptionPricingModelType? ivModel = null, Resolution? resolution = null)
+        {
+            var delta = D(symbol, riskFreeRate, dividendYield, optionModel, ivModel, resolution);
+            delta.SetMirrorOptionContract(mirrorOption);
+            delta.ImpliedVolatility.SetMirrorOptionContract(mirrorOption);
```

Comment:
`Delta/ImpliedVolatility` have a constructor has `mirrorOption` overloads why do we need `SetMirrorOptionContract`, shouldn't it just be a constructor argument like the rest?
Same comment about the `SmoothingFunction`, for normalization, shouldn't it be a constructor argument?